kore/app/parser/Main.hs

```diff
@@ -35,6 +31,10 @@ import Kore.Parser (
  )
 import Kore.Syntax.Definition
 import Kore.Unparser as Unparser
+import Kore.Validate.DefinitionVerifier (
+    verifyAndIndexDefinition,
+ )
+import qualified Kore.Validate.PatternVerifier as PatternVerifier
```

Comment:
We could rename these modules (and everything inside them) from `verifier` to `validator`, but we can do this in a separate PR.